lib/paperclip/version.rb

```diff
@@ -1,3 +1,3 @@
 module Paperclip
-  VERSION = "4.3.7".freeze unless defined? Paperclip::VERSION
+  VERSION = "4.3.6".freeze unless defined? Paperclip::VERSION
```

Comment:
isnt the next version `4.3.8`?

```suggestion
  VERSION = "4.3.8".freeze unless defined? Paperclip::VERSION
```

Reply:
Hah! It turns out I can't increment numbers- must have missed that day in school.

I no longer have access to the original repo I made this PR from, so please feel free to close this PR.